adversarial_workflow/evaluators/runner.py

```
@@ -0,0 +1,313 @@
+    log_file = Path(project_config["log_directory"]) / f"{file_basename}-{config.output_suffix}.md"
+
+    is_valid, verdict, message = validate_evaluation_output(str(log_file))
+
+    if not is_valid:
+        print(f"{RED}Evaluation failed: {message}{RESET}")
+        return 1
+
+    return _report_verdict(verdict, log_file, config)
+
+
+def _report_verdict(verdict: str | None, log_file: Path, config: EvaluatorConfig) -> int:
```

Ruff (0.14.13)

Unused function argument: `config`

(ARG001)